python/lsst/jointcal/dataIds.py

```diff
@@ -59,22 +59,19 @@ def makeDataRefList(self, namespace):
         if self.datasetType is None:
             raise RuntimeError("Must call setDatasetType first")
         skymap = None
-        log = None
+        log = lsst.log.Log.getLogger("jointcal.dataIds")
```

Comment:
That is weird; I thought the logger's name would be `_DefaultName`. 

I agree the `root` logger is not helpful, so do not object your way here.